### DISCUSSION TITLE
FN2 discussion

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/library#341] [General]
An additional activity or two about functions in context of actual things is needed. There is [Activity 2.2.14](https://library.tbil.org/2024/precalculus/FN2.html#activity-fn-application) at the very end, but that's it. I'm thinking something earlier on though to help them translate the words of the application to the notation. Like if we gave them height as a function of time, ask to describe what h(3) means. With choices like "After 3 seconds, the height is 25ft", "The height is 3ft after 25 seconds", "The height is multiplied by 3", etc. So that we are attaching words and context to h(3) and connecting to the fact that h(3) is asking you plug in 3 for time.

**Comment by siwelwerd** [comment]
See #563
